.github/workflows/ci.yml

```diff
@@ -745,3 +748,126 @@ jobs:
           python3 -m pytest -s ./graphscope/tests/kubernetes/test_demo_script.py -k test_demo_on_hdfs
           # Check the result file have successfully written to the given location
           # hdfs dfs -test -e /ldbc_sample/res.csv_0 && hdfs dfs -test -e /ldbc_sample/res.csv_1
+
+  spark-graphx-test:
+    runs-on: [self-hosted, ubuntu2004]
+    if: ${{ github.repository == 'alibaba/GraphScope' }}
+    needs: [build-wheels]
+    steps:
+      - uses: actions/checkout@v2.3.2
+
+      - uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - uses: actions/download-artifact@v2
+        with:
+          path: artifacts
+
+      - name: Add envs to GITHUB_ENV
+        run: |
+          short_sha=$(git rev-parse --short HEAD)
+          echo "SHORT_SHA=${short_sha}" >> $GITHUB_ENV
+
+      - name: Prepare Environment
+        env:
+          GS_TEST_DIR: ${{ github.workspace }}/gstest
+        run: |
+          # download dataset
+          git clone -b master --single-branch --depth=1 https://github.com/7br/gstest.git ${GS_TEST_DIR}
+
+      - name: Setup SSH
+        run: |
+          ssh-keygen -t rsa -f ~/.ssh/id_rsa -N ''
+          cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+          chmod og-wx ~/.ssh/authorized_keys
+          echo "StrictHostKeyChecking no" >> ~/.ssh/config
+          sudo /etc/init.d/ssh start
+
+      - name: Install spark
+        env:
+          GS_TEST_DIR: ${{ github.workspace }}/gstest
+        run: |
+          # download spark
+          wget --no-check-certificate https://dlcdn.apache.org/spark/spark-3.2.2/spark-3.2.2-bin-hadoop2.7.tgz -P /tmp/
+          tar -xzf /tmp/spark-3.2.2-bin-hadoop2.7.tgz -C ~/
+
+          #start spark
+          pushd ~/spark-3.2.2-bin-hadoop2.7/
+          hostname=`hostname`
+          echo ${hostname} > conf/workers
+          ./sbin/start-all.sh
+          popd
+
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
+        if: true
+
+      - name : Run spark test
+        env:
+          GS_TEST_DIR: ${{ github.workspace }}/gstest
+          SAPRK_HOME: ~/spark-3.2.2-bin-hadoop2.7/
```

Comment:
- typo: SPARK
- for the version of Spark, maybe you can contact Siyuan to negotiate a common version? otherwise it is a little wired that GraphScope depends on two versions of Spark.

Reply:
- Changed to `Spark` according to [spark website](https://spark.apache.org/)
- Sure.